examples/Part 01 - The Basics of Private Deep Learning.ipynb

```diff
@@ -0,0 +1,840 @@
+    "\n",
+    "**2) Entrepreneurial Opportunities** - There are a whole host of problems in society that Deep Learning can solve, but many of the most important haven't been explored because it would require access to incredibly sensitive information about people (consider using Deep Learning to help people with mental or relationship issues!). Thus, learning Private Deep Learning unlocks a whole host of new startup opportunities for you which were not previously available to others without these toolsets.\n",
+    "\n",
+    "**3) Social Good** - Deep Learning can be used to solve a wide variety of problems in the real world, but Deep Learning on *personal information* is Deep Learning about people, *for people*. Learning how to do Deep Learning on data you don't own represents more than a career or entrepreneurial opportunity, it is the opportunity to help solve some of the most personal and important problems in people's lives - and to do it at scale.\n",
+    "\n",
+    "## How do I get extra credit?\n",
+    "\n",
+    "- Star PySyft on GitHub! - [https://github.com/OpenMined/PySyft](https://github.com/OpenMined/PySyft)\n",
+    "- Make a Youtube video teaching this notebook!\n",
+    "\n",
+    "\n",
+    "... ok ... let's do this!"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Part -1: Prerequisites\n",
+    "\n",
+    "- Know TensorFlow - if not then take a look at the [TensorFlow tutorials](https://www.tensorflow.org/tutorials/) for more\n",
+    "- Read the PySyft Framework Paper https://arxiv.org/pdf/1811.04017.pdf! This will give you a thorough background on how PySyft is constructed, which will help things make more sense. Note the paper has gotten slightly out of date, so refer to these tutorials or ask in Slack if something seems strange! [slack.openmined.org](http://slack.openmined.org/)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Part 0: Setup\n",
+    "\n",
+    "To begin, you'll need to make sure you have the right things installed. To do so, head on over to PySyft's readme and follow the setup instructions. TLDR for most folks is.\n",
+    "\n",
+    "- Install Python 3.6 or higher\n",
+    "- Install TensorFlow 2.0.0 (`pip install tensorflow==2.0.0`)\n",
+    "- TODO syft-tf install instructions\n",
+    "\n",
+    "If any part of this doesn't work for you (or any of the tests fail) - first check the [README](https://github.com/OpenMined/PySyft.git) for installation help and then open a GitHub Issue or ping the #beginner channel in our slack! [slack.openmined.org](http://slack.openmined.org/)"
```

Comment:
where? in PySyft-TF or main PySyft

Reply:
PySyft-TF, although people will inevitably open things in both places regardless :)

Reply:
wanted to leave it up to the user, but I can include a link

Reply:
in fact, it might make more sense for certain cases to open an issue in PySyft (if it's the root cause). I guess you can leave it up the user. we can address if we see a pb.

Reply:
included a link, we can always open issues in pysyft when we investigate further

Reply:
Sounds good to me! thanks